### ISSUE TITLE
Model(constraint).solve() always returns False

### ISSUE THREAD

**Issue by zemlyansky** [CPMpy/cpmpy#33]
Some examples use results of the `solve` method to conditionally print variable values. (e.g. `if Model(constraints).solve(): print(puzzle.value())`). That method always returns `False` even when trying very simple problems:
 
```
from cpmpy import *

a = IntVar(0, 9)
constraints = [a > 5, a < 7]
print(Model(constraints).solve())
print(a.value())
```

Produces:
```
False
6
```

Tested on:
```
Python 3.7.4
cpmpy==0.6.0
```

**Comment by tias** [comment]
Hi,

The README and ReadTheDocs are ahead of the pip release. If you install from github directly this will not be an issue anymore.

We will release a new version soon, sorry for the inconvenience.


Kind regards,
Tias

**Comment by tias** [comment]
Fixed since 0.7.1 and doing regular updates to keep in sync with master since then
